cmd/service_bootstrap_desktop.go

```diff
@@ -1,7 +1,7 @@
 // +build darwin windows linux,!android
 
 /*
- * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
```

Comment:
I'm not sure, but we probably not need to change this year in the existing files.